samples/httpbin/httpbin-gateway.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Istio Authors
+#
```

Comment:
cc @frankbu 
Can you please change this to use Istio gateway definition? We are trying to not create dependencies on Ingress

Reply:
I agree with @rshriram that this should use a Gateway, instead of Ingress. Also, I think the gateway deploy and test (steps 3 and 4) should be separated into an optional section at the end, since the main purpose of httpbin is for testing internal requests, so many people will not need the gateway. Also, now that it's creating multiple things, we probably should include cleanup instructions in the README.